docs/source/run/parameters.rst

```diff
@@ -611,6 +558,37 @@ Option: ``from_file``
     Name of the beam to be read in. If an openPMD file contains multiple beams, the name of the beam
     needs to be specified.
 
+SALAME algorithm
+^^^^^^^^^^^^^^^^
+
+HiPACE++ features the Slicing Advanced Loading Algorithm for Minimizing Energy Spread (SALAME) to generate a beam profile that
+automatically loads the wake optimally, i.e., that the initial wakefield is flattened by the charge of the beam. Important note:
```

Comment:
```suggestion
HiPACE++ features the Slicing Advanced Loading Algorithm for Minimizing Energy Spread (SALAME) to generate a beam profile that
automatically loads the wake optimally, i.e., so that the initial wakefield is flattened by the charge of the beam. Important note:
```